components/encryption/src/manager/mod.rs

```diff
@@ -121,6 +121,14 @@ impl Dicts {
             // ...else, return either error.
             (file_dict_file, key_bytes) => {
                 if let Err(key_err) = key_bytes {
+                    let (file_dict_file, file_dict) = file_dict_file.unwrap();
```

Comment:
avoid unwrap. In fact we can make a match arm for it ?
```
match (log_content, key_bytes) {
    ...
    (Ok(file_dict_file, Err(key_bytes)) if file_dict_file.0.files.is_empty() => {
        ... // delete empty file dict
    }
    // ...else, return either error
    (file_dict_file, key_bytes) => {
         ...
    }
}
```
(I might be using incorrect syntax though)